plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogConnectorSmokeTest.java

```diff
@@ -217,6 +217,15 @@ protected String getMetadataLocation(String tableName)
                 .getParameters().get("metadata_location");
     }
 
+    @Override
+    protected void deleteDirectory(String location)
+    {
+        AmazonS3 s3 = AmazonS3ClientBuilder.standard().build();
```

Comment:
You may want to use a method similar to https://github.com/trinodb/trino/blob/71a5b05672e3a8dcef855703662079ec7e95f08b/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogConnectorSmokeTest.java#L97-L113

because currently there is nothing actually deleted from S3.